{"address":"192.168.1.100","code":"EHOSTUNREACH","errno":-65,"level":"error","message":"MQTT error connect EHOSTUNREACH 192.168.1.100:1883 - Local (192.168.1.12:56532)","port":1883,"service":"vision-backend","stack":"Error: connect EHOSTUNREACH 192.168.1.100:1883 - Local (192.168.1.12:56532)\n    at internalConnect (node:net:1098:16)\n    at defaultTriggerAsyncIdScope (node:internal/async_hooks:464:18)\n    at node:net:1328:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)","syscall":"connect","timestamp":"2025-09-11T12:06:25.639Z"}
{"level":"error","message":"Gemini analysis failed [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash:generateContent: [503 Service Unavailable] The model is overloaded. Please try again later.","service":"vision-backend","stack":"Error: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash:generateContent: [503 Service Unavailable] The model is overloaded. Please try again later.\n    at handleResponseNotOk (/Users/adndax/Documents/Academic/4thSemester/Lomba/A-Eyes/backend/node_modules/@google/generative-ai/dist/index.js:403:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async makeRequest (/Users/adndax/Documents/Academic/4thSemester/Lomba/A-Eyes/backend/node_modules/@google/generative-ai/dist/index.js:376:9)\n    at async generateContent (/Users/adndax/Documents/Academic/4thSemester/Lomba/A-Eyes/backend/node_modules/@google/generative-ai/dist/index.js:819:22)\n    at async AIService.analyzeImage (/Users/adndax/Documents/Academic/4thSemester/Lomba/A-Eyes/backend/services/ai.js:87:22)\n    at async testGeminiWithLocalImages (/Users/adndax/Documents/Academic/4thSemester/Lomba/A-Eyes/backend/test-gemini.js:36:24)","status":503,"statusText":"Service Unavailable","timestamp":"2025-09-11T12:33:52.647Z"}
